WcaOnRails/app/models/competition.rb

```diff
@@ -1965,4 +1965,10 @@ def find_round_for(event_id, round_type_id, format_id = nil)
         (format_id.nil? || format_id == r.format_id)
     end
   end
+
+  def dues_per_competitor_in_usd
+    DuesCalculator.dues_per_competitor_in_usd(self.country_iso2, self.base_entry_fee_lowest_denomination, self.currency_code)
+  rescue StandardError
```

Comment:
Why are you catching an error here? When could an error potentially be thrown?

Reply:
The case where the currency is not available in Ruby money.

Reply:
In that case, think about more clever ways to handle this edge case. Investigate whether there is a `currency_available?` method in the Money gem that we use.

Catching errors to return a default value is bad practice, and catching generic `StandardError` is very bad practice :see_no_evil: